tests/datasets/test_dataset.py

```diff
@@ -1092,9 +1287,9 @@ def test_normalize_timestamps_raises_value_error_for_invalid_input() -> None:
             dataframe=DataFrame(
                 {
                     "timestamp": [
-                        "2023-03-24 10:00:00 UTC",
-                        "2023-03-24 11:30:00 UTC",
-                        "2023-03-24 14:15:00 UTC",
+                        "24-03-2023 invalidCharacter 14:15:00",
+                        "24-03-2023 invalidCharacter 10:30:00",
+                        "24-03-2023 invalidCharacter 18:45:00",
```

Comment:
Mistaken checkin? `invalidCharacter`

Reply:
this test is for raising exceptions on invalid entries, so this is fine